### ISSUE TITLE
Inference Score is low AP - on different dataset

### ISSUE THREAD

**Issue by Mahmood-Hussain** [SwinTransformer/Swin-Transformer-Object-Detection#197]
Hello I tried the model and AP is above 40 percent but when I test it on a different dataset the AP is around 0.04 what could be the possible reasons for failure? 
I trained the model on 256x256 image size and tested on 256x256 and higher sizes but nothing changed AP is still very low on the testing set. Even during I have a separate validation set but AP is good enough around 40

**Comment by Banks1y** [comment]
I also have the same problem. I think it may be caused by overfitting of the model. Have you solved this problem？
